shell/browser/web_dialog_helper.cc

```diff
@@ -331,6 +338,7 @@ void WebDialogHelper::EnumerateDirectory(
     content::WebContents* web_contents,
     std::unique_ptr<content::FileSelectListener> listener,
     const base::FilePath& dir) {
+  LOG(INFO) << "WebDialogHelper::EnumerateDirectory";
```

Comment:
Can you remove this log ?

Reply:
oops, yes!

Reply:
done, & updated it to make it slightly more robust in the case that `listener_` was not initialized